package.json

```diff
@@ -9,8 +9,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "tsc --noEmit && eslint '*/**/*.{js,ts,tsx}' --quiet",
-    "start": "webpack-serve --config example/webpack.config.js",
+    "start": "webpack-serve --config example/webpack.config.js --port 9000",
```

Comment:
default port is quite overloaded. would there be a better default port to pick. I chose randomly here